aws-examples/mistral-e2e/chat.py

```diff
@@ -0,0 +1,62 @@
+# This is a simple command-line chat application that has contextual memory.
+
```

Comment:
Can you add a licence here (as well as the other python files) please? You can check our examples.